commit d9fcfb60a7096358ea45bff03da3e21c05ea6b39 (refs/changes/64/864/3)
Author: Patrick Mooney <pmooney@pfmooney.com>
Date:   2016-11-09T22:27:26-06:00 (2 years, 11 months ago)
    
    PUBAPI-1346 allow ufds_master credentials to differ from local ufds
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/sapi_manifests/cloudapi/template b/sapi_manifests/cloudapi/template
index 34fd725..9e9f947 100644
--- a/sapi_manifests/cloudapi/template
+++ b/sapi_manifests/cloudapi/template
@@ -28,7 +28,12 @@
     "ufds_master": {
         "url": "ldaps://{{{ufds_remote_ip}}}",
         "bindDN": "{{{ufds_ldap_root_dn}}}",
+        {{^ufds_remote_ldap_root_pw}}
         "bindPassword": "{{{ufds_ldap_root_pw}}}",
+        {{/ufds_remote_ldap_root_pw}}
+        {{#ufds_remote_ldap_root_pw}}
+        "bindPassword": "{{{ufds_remote_ldap_root_pw}}}",
+        {{/ufds_remote_ldap_root_pw}}
         "cache": {
             "size": 5000,
             "age": 60
